src/ui/component/claimList/view.jsx

```diff
@@ -92,6 +95,7 @@ export default function ClaimList(props: Props) {
         <div className={classnames('claim-list__header', { 'claim-list__header--small': type === 'small' })}>
           {header}
           {loading && <Spinner type="small" />}
+          {obscureNsfw && <HiddenNsfw type="page" />}
```

Comment:
I'm not sure this should live in this component. I don't think this should be shown on the home page or your list of publishes. Probably only in `page/tags/view.jsx`, maybe we could pass a prop down from there?

I like the styling on this though!

Reply:
Moved it to the tags view! Also I edited the icons .scss to give a little bit of space between the icon and the message which are mashed together now that its in the I18nMessage component.